Vidyano.Core/ViewModel/PersistentObjectAttribute.cs

```diff
@@ -59,7 +59,12 @@ internal string[] OptionsDirect
             set
             {
                 if (OptionsDirect != null && value != null && OptionsDirect.SequenceEqual(value))
+                {
+                    if (Client.StrictMode)
+                        throw new InvalidOperationException("The current value is the same as the specified value.");
```

Comment:
we don't want to throw if they set the same value, as otherwise we need to change all the code to first check which isn't something we normally would do